layers/meta-resin-raspberrypi/recipes-core/images/resin-image.bbappend

```diff
@@ -8,7 +8,9 @@ SDIMG_KERNELIMAGE_raspberrypi3-64 ?= "kernel8.img"
 # Customize resinos-img
 RESIN_IMAGE_BOOTLOADER_rpi = "bcm2835-bootfiles"
 RESIN_BOOT_PARTITION_FILES_rpi = " \
-    ${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin:/${SDIMG_KERNELIMAGE} \
+		u-boot.bin:/${SDIMG_KERNELIMAGE} \
+		boot.scr:/boot.scr \
+		${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin:/zImage \
```

Comment:
Shouldn't zImage be SDIMG_KERNELIMAGE?

Reply:
`SDIMG_KERNELIMAGE` is actually `kernel.img` `kernel7.img` or `kernel8.img`

But I guess that zImage should be `KERNEL_IMAGETYPE` which is zImage in the local.conf.sample file

Reply:
Yup. I actually meant KERNEL_IMAGETYPE.